### ISSUE TITLE
4-projects shared step fails with `error: string validation failed <PROJECT-ID>_cloudbuild`

### ISSUE THREAD

**Issue by daniel-cit** [terraform-google-modules/terraform-example-foundation#770]
### TL;DR

4-projects shared step fails with `error: string validation failed <PROJECT-ID>_cloudbuild`

### Expected behavior

4-projects shared step run without error

### Observed behavior

step 4-projects shared fails with error:

```
error: string validation failed <PROJECT-ID>_cloudbuild
  with module.infra_pipelines.google_storage_bucket.pipeline_infra["cloudbuild"],
  on ../../modules/infra_pipelines/main.tf line 49, in resource "google_storage_bucket" "pipeline_infra":
  49: resource "google_storage_bucket" "pipeline_infra" {
```

### Terraform Configuration

```hcl
default configuration at 22022-08-04
```

### Terraform Version

```sh
Terraform v0.13.7
```

### Additional information

there is a fix for this error https://github.com/hashicorp/terraform-provider-google/pull/12248 to enter a new release

A work around is to configure the version on the infra pipeline module to avoid the version with and issue until the fix is in a new release

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/4-projects/modules/infra_pipelines/versions.tf
```
    google = {
      source  = "hashicorp/google"
      version = ">= 3.50, <= 4.30.0"
    }
    google-beta = {
      source  = "hashicorp/google-beta"
      version = ">= 3.50, <= 4.30.0"
    }
```
